contributing/code/standards.rst

```diff
@@ -218,7 +218,8 @@ Service Naming Conventions
 
 * The DI alias of the bundle is the first group (e.g. ``fos_user``);
 
-* Use lowercase letters for service and parameter names;
+* Use lowercase letters for service and parameter names (except when referring
+  to environment variables with the ``%env(VARIABLE_NAME)%`` syntax);
```

Comment:
Maybe we should note that they should not begin with an underscore as it is to be deprecated.